package.json

```diff
@@ -50,7 +50,6 @@
     "test:debug": "lerna run test:debug --stream",
     "typecheck": "tsc",
     "update-deps": "lerna exec '../../node_modules/.bin/ncu --upgrade' && lerna exec 'npm i --package-lock-only' && npm run reinstall-deps",
-    "verify": "lerna run verify --stream",
```

Comment:
_question:_ What's up with this deletion?

Reply:
This was a mistake.